src/providers/sdk/__tests__/Base.test.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * @jest-environment jsdom
+ */
+import { CryptKeeperInjectedProvider } from "..";
+
+jest.mock("@src/background/services/event", (): unknown => ({
+  __esModule: true,
+  default: jest.fn().mockImplementation(),
+}));
+
+jest.mock("@src/background/services/zkProof", (): unknown => ({
+  __esModule: true,
+  default: {
+    getInstance: jest.fn(),
+  },
+}));
+
+jest.useRealTimers();
+
+const createTimeout = (): Promise<void> =>
+  new Promise((resolve) => {
+    // need to wait until we get popup and add request to queue
+    setTimeout(resolve, 10000);
```

Comment:
Adding timeouts in tests is not a good practice. It can be a cause of flaky tests and memory leaks.